phenopy/util.py

```diff
@@ -1,9 +1,16 @@
 import csv
+import os
 import sys
 import networkx as nx
 import pandas as pd
 
-from phenopy.config import logger
+from collections import Counter
+
+from phenopy.config import logger, project_data_dir
+
+
+# path to phenotype groups
+PHENOTYPE_GROUP_FILE = os.path.join(project_data_dir, "phenotype_groups.txt")
```

Comment:
same as above here. This belongs in config.

Reply:
done